cloudformation/adx-data-plane.cfn.yaml

```diff
@@ -0,0 +1,245 @@
+              - dataexchange:GetDataSet
+              - dataexchange:ListDataSets
+              - dataexchange:UpdateRevision
+              - dataexchange:ListJobs
+              - dataexchange:GetJob
+              - dataexchange:CreateAsset
+              - dataexchange:GetRevision
+              - dataexchange:CreateJob
+              - dataexchange:CreateRevision
+              - dataexchange:UpdateDataSet
+              - dataexchange:UpdateAsset
+              - dataexchange:StartJob
+              - dataexchange:ListDataSetRevisions
+            Effect: Allow
+            Resource: "*"
+          - Action:
+              - aws-marketplace:ListChangeSets
+              - aws-marketplace:DescribeChangeSet
+              - aws-marketplace:StartChangeSet
+              - aws-marketplace:CancelChangeSet
+              - aws-marketplace:ListEntities
+              - aws-marketplace:DescribeEntity
+              - aws-marketplace:ListTasks
+              - aws-marketplace:DescribeTask
+              - aws-marketplace:UpdateTask
+              - aws-marketplace:CompleteTask
+            Effect: Allow
+            Resource: "*"
+          - Action:
+              - s3:GetObject
+              - s3:ListBucket
+            Condition:
+              ForAnyValue:StringEquals:
+                aws:CalledVia: dataexchange.amazonaws.com
+            Effect: Allow
+            Resource:
+              - Fn::Join:
+                  - ""
+                  - - "arn:"
+                    - Ref: AWS::Partition
+                    - ":s3:::"
+                    - Ref: AssetBucketName
+              - Fn::Join:
+                  - ""
+                  - - "arn:"
+                    - Ref: AWS::Partition
+                    - ":s3:::"
+                    - Ref: AssetBucketName
+                    - /*
+          - Action:
+              - s3:GetObject
+              - s3:ListBucket
+              - s3:PutObject
+              - s3:PutObjectAcl
+            Condition:
+              ForAnyValue:StringEquals:
+                aws:CalledVia: dataexchange.amazonaws.com
+            Effect: Allow
+            Resource: arn:aws:s3:::*aws-data-exchange*
+        Version: "2012-10-17"
+      PolicyName: CrossAccountRoleDefaultPolicy212A317F
+      Roles:
+        - Ref: CrossAccountRoleFACE29D1
+    Metadata:
+      aws:cdk:path: ADXDataPlaneStack/CrossAccountRole/DefaultPolicy/Resource
+  RegistrationCustomResource:
+    Type: AWS::CloudFormation::CustomResource
+    Properties:
+      ServiceToken:
+        Ref: RegistrationTopicARN
+      ControlPlaneRoleArn:
+        Fn::GetAtt:
+          - CrossAccountRoleFACE29D1
+          - Arn
+      ControlPlaneExternalId:
+        Ref: ExternalId
+      AssetBucketName:
+        Ref: AssetBucketName
+      CustomerID:
+        Ref: AWS::AccountId
+    UpdateReplacePolicy: Delete
+    DeletionPolicy: Delete
+    Metadata:
+      aws:cdk:path: ADXDataPlaneStack/RegistrationCustomResource/Default
+  CDKMetadata:
```

Comment:
Should we opt out of version reporting? or what's the benefit of having this Metadata?